x-pack/plugins/enterprise_search/server/routes/enterprise_search/engines.ts

```diff
@@ -130,7 +130,7 @@ export function registerEnginesRoutes({ log, router }: RouteDependencies) {
     elasticsearchErrorHandler(log, async (context, request, response) => {
       const { client } = (await context.core).elasticsearch;
       const engines = await client.asCurrentUser.transport.request<SearchResponse>({
-        body: {},
+        body: request.body,
```

Comment:
This should really be using `search` [method](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#_search) instead of using the transport.request.